shell/lib/db-deprecated.js

```diff
@@ -54,9 +54,11 @@ matchWildcardHost = globalDb.matchWildcardHost.bind(globalDb);
 makeWildcardHost = globalDb.makeWildcardHost.bind(globalDb);
 allowDevAccounts = globalDb.allowDevAccounts.bind(globalDb);
 roleAssignmentPattern = globalDb.roleAssignmentPattern;
+addUserActions = globalDb.addUserActions.bind(globalDb);
```

Comment:
Don't add new deprecated methods here -- instead, use `globalDb` directly at the call site.